src/rastervision/protos/chain_workflow.proto

```diff
@@ -18,6 +18,13 @@ message ChainWorkflowConfig {
     // during training.
     repeated Scene test_scenes = 18;
 
+    // The predict_scenes are not associated with any ground truth labels and
+    // are only used for making predictions to visually inspect. These scenes
+    // should contain an id, a raster_source and a ground_truth_label_source
+    // that has a blank URI. (The ground_truth_label_source is still needed
+    // to infer the type of the prediction file that should be generated.)
+    repeated Scene predict_scenes = 20;
```

Comment:
@lewfish what is the significance of `20` here? what does this line do?

Reply:
just so I understand this a bit better

Reply:
In protobufs, each field has to have a unique id that's supposed to be stable over time as the protobuf schema changes. It has something to do with making protobufs backwards compatible. 20 is just an id that hasn't been used yet by other fields.

Reply:
ah, that makes sense